manifests/server.pp

```diff
@@ -392,9 +392,10 @@
 #   Check https://man.openbsd.org/sshd_config#UseDNS for possible values.
 #
 # @param use_pam
-#   TODO: missing in docs
-#   Value(s) passed to  parameter in ssh_config. Unused if empty.
-#   Check  for possible values.
+#   Value(s) passed to UsePAM parameter in ssh_config. Unused if empty.
+#   Possible values are 'yes' and 'no'.
+#   There is no mentioning of this parameter in the current man pages of OpenSSH v7.
+#   But it is mentioned in the release notes of OpenSSH v8. https://www.openssh.com/txt/release-8.0
```

Comment:
It is rather strange that this parameter isn't mentioned in the man pages at all. Since most OSes do support and use it. OpenSSH is first mentioning it in the release notes of v8.

Because most OSes this module does support use it, I have left it here to avoid to confuse users like me now ;)